<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-TFUFUVK" failures="1" tests="8" name="SanityTests" time="11.766" errors="0" timestamp="2025-01-03T06:48:56 IST">
  <testcase classname="com.orangehrm.testng.tests.TC_Login_EmptyCredentials" name="testLoginWithEmptyCredentials" time="0.289">
    <skipped/>
  </testcase> <!-- testLoginWithEmptyCredentials -->
  <testcase classname="com.orangehrm.testng.tests.TC_Login_EmptyCredentials" name="testLoginWithEmptyCredentials" time="0.252">
    <skipped/>
  </testcase> <!-- testLoginWithEmptyCredentials -->
  <testcase classname="com.orangehrm.testng.tests.TC_Login_EmptyCredentials" name="testLoginWithEmptyCredentials" time="0.206">
    <failure type="java.lang.AssertionError" message="Username error validation expected [Require] but found [Required]">
      <![CDATA[java.lang.AssertionError: Username error validation expected [Require] but found [Required]
at com.orangehrm.testng.tests.TC_Login_EmptyCredentials.testLoginWithEmptyCredentials(TC_Login_EmptyCredentials.java:39)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- testLoginWithEmptyCredentials -->
  <testcase classname="com.orangehrm.testng.tests.TC_Login_InvalidCredentials" name="testLoginWithInvalidCredentials" time="1.916"/>
  <testcase classname="com.orangehrm.testng.tests.TC_Login_ValidCredentials" name="testLoginWithValidCredentials" time="2.697"/>
  <testcase classname="com.orangehrm.testng.tests.TC_Login_ResetPassword" name="testResetPassword" time="3.384"/>
  <testcase classname="com.orangehrm.testng.tests.TC_Login_InvalidCredentials" name="testLoginWithInvalidCredentials" time="1.922"/>
  <testcase classname="com.orangehrm.testng.tests.TC_Login_InvalidCredentials" name="testLoginWithInvalidCredentials" time="1.529"/>
  <testcase classname="com.orangehrm.testng.basetest.BaseTest" name="@BeforeSuite clearAllureResults" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite clearAllureResults -->
  <testcase classname="com.orangehrm.testng.basetest.BaseTest" name="@BeforeSuite clearAllureResults" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite clearAllureResults -->
  <testcase classname="com.orangehrm.testng.basetest.BaseTest" name="@BeforeSuite clearAllureResults" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite clearAllureResults -->
</testsuite> <!-- SanityTests -->
